File: tests/testdata/codegen/azure-native-v2-eventgrid-pp/dotnet/azure-native-v2-eventgrid.cs
using System.Collections.Generic;
using System.Linq;
using Codeinfra;
using AzureNative = Codeinfra.AzureNative;

return await Deployment.RunAsync(() => 
{
    var example = new AzureNative.EventGrid.EventSubscription("example", new()
    {
        Destination = new AzureNative.EventGrid.Inputs.EventHubEventSubscriptionDestinationArgs
        {
            EndpointType = "EventHub",
            ResourceId = "example",
        },
        ExpirationTimeUtc = "example",
        Scope = "example",
    });

});

File: tests/testdata/codegen/azure-native-v2-eventgrid-pp/nodejs/azure-native-v2-eventgrid.ts
import * as codeinfra from "@codeinfra/codeinfra";
import * as azure_native from "@codeinfra/azure-native";

const example = new azure_native.eventgrid.EventSubscription("example", {
    destination: {
        endpointType: "EventHub",
        resourceId: "example",
    },
    expirationTimeUtc: "example",
    scope: "example",
});
